test/e2e/kuma_deploy_hybrid_test.go

```diff
@@ -13,7 +13,7 @@ import (
 	. "github.com/kumahq/kuma/test/framework"
 )
 
-var _ = Describe("Test Kubernetes/Universal deployment", func() {
+var _ = XDescribe("Test Kubernetes/Universal deployment", func() {
```

Comment:
why `X`?

Reply:
This one will break, we need the bi-directional KDS to make this work.
Simply - put, Universal is run from memory so resources do not persist during restarts, and we have no way to use the Zone resources unless we restart (which will erase the resource from memory)